modules/kernel/src/main/scala/tofu/Scoped.scala

```diff
@@ -92,11 +94,11 @@ trait ScopedInstances {
   final def makeExecute[Tag, F[_]](p1: ExecutionContext): ScopedExecute[Tag, F] =
     macro Interop.delegate1p1[Execute[F], Tag, F, { val `tofu.interop.CE2Kernel.makeExecute`: Unit }]
 
-  final implicit def asyncExecute[F[_]]: Execute[F] =
-    macro Interop.delegate[Execute[F], F, { val `tofu.interop.CE2Kernel.asyncExecute`: Unit }]
+  final def makeExecute3[Tag, F[_]](p1: ExecutionContext): ScopedExecute[Tag, F] =
```

Comment:
Inconsistent naming with `makeExecute`

Reply:
I wonder if we need to rename existing for some amount of versions method `makeExecute` for cats-effect 2